libs/blocks/figure/figure.js

```diff
@@ -1,4 +1,4 @@
-import { applyHoverPlay, decorateAnchorVideo } from '../../utils/decorate.js';
+import { applyHoverPlay, decorateAnchorVideo, applyAccessibiltyEvents, decoratePausePlayWrapper } from '../../utils/decorate.js';
```

Comment:
Why are the `applyAccessibiltyEvents, decoratePausePlayWrapper` exported within `decorate.js` when they are only ever used once in the figure block?

Reply:
They are used in decorate.js L:486 as well.

Reply:
They are used within `decorate.js` as well L: 484 and L:486.